code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -787,4 +720,18 @@ mob/living/simple_animal/bullet_act(var/obj/item/projectile/Proj)
 			adjustFireLoss(rand(3, 5))
 
 /mob/living/simple_animal/get_digestion_product()
-	return "nutriment"
\ No newline at end of file
+	return "nutriment"
+
+/mob/living/simple_animal/bullet_impact_visuals(var/obj/item/projectile/P, var/def_zone, var/damage)
```

Comment:
I am fairly sure that some simple mobs would not have blood, such as cult creatures, or would bleed oil, like some of the robots.

Reply:
fixed @Alberyk